stage1/init/init.go

```diff
@@ -369,6 +584,18 @@ func stage1() int {
 		return 3
 	}
 
+	// The systemd version shipped with CoreOS (v215) doesn't allow the
```

Comment:
can you add a big TODO here please? (at some point I want to modify the test script to actually audit TODOs in the codebase and mandate that they have end dates)